pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/StartServiceManagerCommand.java

```diff
@@ -0,0 +1,213 @@
+  @Option(name = "-clusterName", required = true, metaVar = "<String>", usage = "Pinot cluster name.")
+  private String _clusterName = DEFAULT_CLUSTER_NAME;
+  @Option(name = "-port", required = true, metaVar = "<int>", usage = "Pinot service manager admin port, -1 means disable, 0 means a random available port.")
+  private int _port;
+  @Option(name = "-bootstrapConfigPaths", handler = StringArrayOptionHandler.class, required = false, usage = "A list of Pinot service config file paths. Each config file requires an extra config: 'pinot.service.role' to indicate which service to start.", forbids = {"-bootstrapServices"})
+  private String[] _bootstrapConfigPaths;
+  @Option(name = "-bootstrapServices", handler = StringArrayOptionHandler.class, required = false, usage = "A list of Pinot service roles to start with default config. E.g. CONTROLLER/BROKER/SERVER", forbids = {"-bootstrapConfigPaths"})
+  private String[] _bootstrapServices = BOOTSTRAP_SERVICES;
+
+  private PinotServiceManager _pinotServiceManager;
+
+  public String getZkAddress() {
+    return _zkAddress;
+  }
+
+  public StartServiceManagerCommand setZkAddress(String zkAddress) {
+    _zkAddress = zkAddress;
+    return this;
+  }
+
+  public String getClusterName() {
+    return _clusterName;
+  }
+
+  public StartServiceManagerCommand setClusterName(String clusterName) {
+    _clusterName = clusterName;
+    return this;
+  }
+
+  public int getPort() {
+    return _port;
+  }
+
+  public StartServiceManagerCommand setPort(int port) {
+    _port = port;
+    return this;
+  }
+
+  public String[] getBootstrapConfigPaths() {
+    return _bootstrapConfigPaths;
+  }
+
+  public StartServiceManagerCommand setBootstrapConfigPaths(String[] bootstrapConfigPaths) {
+    _bootstrapConfigPaths = bootstrapConfigPaths;
+    return this;
+  }
+
+  public String[] getBootstrapServices() {
+    return _bootstrapServices;
+  }
+
+  public StartServiceManagerCommand setBootstrapServices(String[] bootstrapServices) {
+    _bootstrapServices = bootstrapServices;
+    return this;
+  }
+
+  @Override
+  public boolean getHelp() {
+    return _help;
+  }
+
+  public void setHelp(boolean help) {
+    _help = help;
+  }
+
+  @Override
+  public String getName() {
+    return "StartPinotService";
+  }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder()
+        .append("StartServiceManager -clusterName " + _clusterName + " -zkAddress " + _zkAddress + " -port " + _port);
+    if (_bootstrapConfigPaths != null) {
+      sb.append(" -bootstrapConfigPaths " + Arrays.toString(_bootstrapConfigPaths));
+    } else if (_bootstrapServices != null) {
+      sb.append(" -bootstrapServices " + Arrays.toString(_bootstrapServices));
+    }
+
+    return sb.toString();
+  }
+
+  @Override
+  public void cleanup() {
+  }
+
+  @Override
+  public String description() {
+    return "Start the Pinot Service Process at the specified port.";
```

Comment:
this description is not very clear. "Starts the PinotServiceManager..." ?

Reply:
Yes. Let's please add documentation here describing the usage of this command

Reply:
added